### ISSUE TITLE
Feature idea: extended flyby listings by searching flybys of other listed athletes

### ISSUE THREAD

**Issue by ck-on** [thomaschampagne/elevate#316]
While I doubt it is easily possible to search for activities around your activity (ie. rides within one hour of yours but not overlapping yours) it occurs to me that by examining the flybys of other athletes in your flybys, you can find even more flybys that don't show up on your listing because they didn't quite overlap yours, but did the other athletes. 

I've checked this idea out and it seems to work but it is tedious.

If stravistix could automate that process, it would be fantastic - you could see some people you missed within an hour or two.

Or maybe even more advanced, take the segments from your activity and somehow search for other people doing that segment that day and then adding them to the flyby.

Basically a "Flyby TODAY" feature

Thanks for considering and keep up the amazing work.
